examples/custom-resource/README.md

```diff
@@ -0,0 +1,35 @@
+## GitHub Knative EventListener
+
+Creates an EventListener that listens for GitHub webhook events.
+
+### Try it out locally:
+
+1. To create the GitHub trigger and all related resources, run:
+
+   ```bash
+   kubectl apply -f examples/custom-resource/
+   ```
+
+1. Test by sending the sample payload.
+
+   ```bash
+   curl -v \
+   -H 'X-GitHub-Event: pull_request' \
+   -H 'X-Hub-Signature: sha1=ba0cdc263b3492a74b601d240c27efe81c4720cb' \
+   -H 'Content-Type: application/json' \
+   -d '{"action": "opened", "pull_request":{"head":{"sha": "28911bbb5a3e2ea034daf1f6be0a822d50e31e73"}},"repository":{"clone_url": "https://github.com/tektoncd/triggers.git"}}' \
+   http://localhost:8080
+   ```
+
+   The response status code should be `201 Created`
```

Comment:
I would avoid "should" - it introduces uncertainty. The users either sees the 201 Created status or they don't. What do they do if they don't see it?